Client/Frontend/UserContent/UserScripts/MainFrame/AtDocumentStart/ReaderMode.js

```diff
@@ -65,7 +65,9 @@ function checkReadability() {
         return;
       }
 
-      var doc = new DOMParser().parseFromString(docStr, "text/html");
+      const DOMPurify = require('dompurify');
+      const clean = DOMPurify.sanitize(docStr);
```

Comment:
By default, this will sanitize the `<body>` of the document only so we might miss vital meta information that is in the `<head>` of the page. 
```suggestion
      const clean = DOMPurify.sanitize(docStr, {WHOLE_DOCUMENT: true});
```